src/actions/pause/extra.js

```diff
@@ -50,9 +66,7 @@ async function getExtraProps(getState, expression, result, evaluate) {
       }
     }
 
-    if (!props.react) {
-      props.react = await getReactProps(evaluate);
-    }
+    merge(props.react, await getReactProps(evaluate));
```

Comment:
can we use splat here `{...props.react, ...await getReactProps(evaluate) }`

Reply:
sure can. 106eeaa17ebc51250d1927ad053372968c2cbacc

Reply:
update: bc60352db7980803b7cded66a2063a30e3a99155